packages/twenty-front/src/modules/favorites/hooks/useWorkspaceFavorites.ts

```diff
@@ -52,5 +54,29 @@ export const useWorkspaceFavorites = () => {
     ],
   );
 
-  return { sortedWorkspaceFavorites };
+  const workspaceFavoriteIds = new Set(
```

Comment:
this part of the code was (and still is) a big "spaghetti". I think here we should have one hook to encapsulate the workspaceFavorite logic instead of having many hooks relying on eachother.

Reply:
the logic is too coupled to be split in different hooks IMO